pkg/provisioner/ironic/ironic.go

```diff
@@ -1050,26 +1034,27 @@ func (p *ironicProvisioner) setUpForProvisioning(ironicNode *nodes.Node, hostCon
 	)
 	p.publisher("ProvisioningStarted",
 		fmt.Sprintf("Image provisioning started for %s", p.host.Spec.Image.URL))
-	return
+	// TODO(zaneb): This doesn't make much sense as the overall operation isn't
+	// complete, but it's currently the only return value that the caller can
+	// distinguish from retryAfterDelay().
+	return operationComplete()
```

Comment:
Are you referring to `actionProvisioning`?

Reply:
I updated this commit to just return here (as it did before) instead of returning operationComplete(), and removed the comment.